extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1038,11 +1038,39 @@ export class AmpLightboxGallery extends AMP.BaseElement {
    *  -Esc will close the lightbox.
    * @private
    */
-  handleKeyboardEvents_(event) {
-    const code = event.keyCode;
-    if (code == KeyCodes.ESCAPE) {
-      this.close_();
+  onKeyDown_(event) {
+    if (!this.isActive_) {
+      return;
+    }
+    const {keyCode} = event;
+    switch (keyCode) {
+      case KeyCodes.ESCAPE:
+        this.close_();
+        return;
+      case KeyCodes.LEFT_ARROW:
+        this.maybeSlideCarousel_(/*Prev*/ -1);
+        return;
+      case KeyCodes.RIGHT_ARROW:
+        this.maybeSlideCarousel_(/*Next*/ 1);
+        return;
+      default:
```

Comment:
I believe you don't need a default return unless the linter complains.

If so then just fallback to `if (keyCode == WHATEVER) { ... return; }` for simplicity.

Reply:
Yeah I forgot. The proper use of switch statements in this case is to use `break;` not `return;`...